.codex/skills/rust-error-handling/SKILL.md

```
@@ -0,0 +1,131 @@
+---
+name: rust-error-handling
+description: Rustでのエラー設計を、境界ごとに thiserror / anyhow を使い分けて実装する。ドメイン/ライブラリは型付きエラー(thiserror)、アプリ境界のみ anyow。context付与、unwrap禁止、HTTP/CLI変換の指針を含む。
+---
+
+# Rust Error Handling: anyhow / thiserror の境界設計
+
+## 概要
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)